### PR TITLE
Report display

### LINKED ISSUE #61: Display initial report
The report should contain the information that  is returned from the API

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
- Display final report
- Add missing styling
- Installs `moment.js`
### Where should the reviewer start?
- `src/components/Report/Report.tsx`
- `src/components/ReportCategory/ReportCategory.tsx`

### How should this be manually tested?
- run `npm install` to install missing dependencies
- navigate to the `generate_report` route with `http://localhost:3001/generate_report`
- make sure to run `npm install -g json-server`
- create a db.json
- paste the data into the newly created db.json file with the following format
```
{
  "posts": [
    { "id": 1, "title": "json-server", "author": "typicode" }
  ],
  "comments": [
    { "id": 1, "body": "some comment", "postId": 1 }
  ],
  "profile": { "name": "typicode" }
}
```
- `json-server --watch db.json`

### What are the relevant tickets?
closes #61 
### Screenshots (if appropriate)
Screen Shot 2021-01-11 at 1 48 59 PM

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -35,20 +47,26 @@ const App:React.FC = () =>{
...
-          <Route exact path="/home" component={Home}/>
-          <Route exact path="/journey" component={Journey}/>
-          <Route exact path="/survey" component={Survey}/>
-          <Route exact path="/question" component={()=><Question
-             updateAllAnswers={updateAllAnswers}/>}/>
-          <Route exact path="/generate_report" component={GenerateReport}/>
-          <Route exact path="/submit" GenerateReport/>
-          <Route exact path="/report" component={Report} />
-          <Route path='/*' component={Error}/>
-        </Switch>
```
Comment (BlakeDonn) [inline src/components/App/App.tsx]: Just a note, we may need to adjust this pathing to make home our default page. We will have to test if it currently loads to the 404 page or home on launch

File: src/components/GenerateReport/GenerateReport.tsx [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
 
-const GenerateReport = () => {
+interface Props{
+  updateReport: any;
+}
+
+const GenerateReport: React.FC<Props> = ({ updateReport}) => {
+  const history = useHistory()
+  const report = useContext(ReportContext)
+
```
Comment (timkeresey) [inline src/components/GenerateReport/GenerateReport.tsx:16]: Api call will go here when we get endpoints

**Review by timkeresey — APPROVED** [review]

File: src/components/Report/Report.tsx [hunk 3]
```diff
@@ -1,84 +1,162 @@
...
+            || userReport[data].monthly_principal
+            || userReport[data].down_payment_percentage_selected
+          }
+          categoryMainTitle={subtitle_1}
+          categorySubtitle={subtitle_2}
+          categorySecondNumber={
+            userReport[data].location
+            || userReport[data].goal_principal
+            || userReport[data].estimated_true_monthly
+            || userReport[data].down_payment_saved
```
Comment (BlakeDonn) [inline src/components/Report/Report.tsx:57]: Wish there was a way we could clean this up, but I think this is the only way we can achieve this without adjusting our data model on the backend.

**Review by BlakeDonn — APPROVED** [review]
Looks good to me, thanks for the hard work on tinkering with the report.

**Pull request merged**